### PR TITLE
(Breaking changes)Scp914 events rework

### PR DESCRIPTION
**Pull request by IRacle1**

### CODE REVIEW

**Comment by IRacle1** [comment]
Plz, add a "do not merge"

**Comment by joker-119** [comment]
I'm not certain the eventargs renaming is necessary, seeing as it's in an event in the Events.Handlers.Scp914 class already.
But this being a breaking change means it won't be merged for a decently long time.

File: Exiled.Events/Patches/Events/Scp914/InteractingEvents.cs [hunk 1]
```diff
@@ -16,51 +19,93 @@ namespace Exiled.Events.Patches.Events.Scp914
...
+                new(OpCodes.Callvirt, Method(typeof(Handlers.Scp914), nameof(Handlers.Scp914.OnActivating))),
+                new(OpCodes.Callvirt, PropertyGetter(typeof(ActivatingScp914EventArgs), nameof(ActivatingScp914EventArgs.IsAllowed))),
+                new(OpCodes.Brfalse_S, returnLabel),
+            });
+
+            newInstructions[newInstructions.Count - 1].labels.Add(returnLabel);
+
+            for (int z = 0; z < newInstructions.Count; z++)
+            {
+                Log.Debug(newInstructions[z].opcode + " " + newInstructions[z].operand);
```
Comment (xRoier) [inline Exiled.Events/Patches/Events/Scp914/InteractingEvents.cs:99]: You forgot to delete those debug lines (Joker 2.0?)

Comment (IRacle1) [reply Exiled.Events/Patches/Events/Scp914/InteractingEvents.cs:99]: Oh, yes) i remove it soon

**Comment by Thundermaker300** [comment]
> I'm not certain the eventargs renaming is necessary, seeing as it's in an event in the Events.Handlers.Scp914 class already. But this being a breaking change means it won't be merged for a decently long time.

I personally believe this is necessary. It can cause confusion otherwise. Imagine if Scp330 handler had an "Interacting" event and an "InteractingEventArgs" ev.

**Pull request closed without merging**